src/content/enterprise/aks.mdx

```diff
@@ -28,10 +28,10 @@ This guide will assume that your domain is registered in [AzureDNS](https://docs
 ### Deploy a Kubernetes cluster
 
 We recommend that you follow Azure's [cluster creation guide](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough).
-Okteto supports Kubernetes versions 1.20 through 1.22.
+Okteto supports Kubernetes versions 1.21 through 1.23.
```

Comment:
Checkr and Dafiti are on < 1.21, what is the plan for them @jmacelroy @pchico83 ? Are they aware of this?

Reply:
I will coordinate with them, but I would say they cannot upgrade to 1.0 if they are not in the right kubernetes version

Reply:
Those are two of our most developed customers, we can't tell them "upgrade or else". They are on premium support, so we need to talk to them so they are aware of our plan before we publish this.

Reply:
@rberrelleza I think requiring a kubernetes upgrade is the right choice here and what other players (like nginx controller) are doing. Maintaining anything under 1.21 will make it much harder to support 1.25.
I want to coordinate with every self-hosted customer the upgrade to 1.0, to make it clear that there are breaking changes and that k8s >= 1.21 is required. Sounds reasonable to me given the fact this is a mayor release

Reply:
@rberrelleza yes, they are aware of this, but I will sync with them again things get publcished. This change applies to the installation guides of new clusters